lua/starfall/libs_sh/mesh.lua

```diff
@@ -699,6 +702,16 @@ if CLIENT then
 		return meshes
 	end
 
+	--- Creats a mesh without any vertex data.
+	-- @return Mesh object
+	-- @client
+	function mesh_library.createEmpty()
+		checkpermission(instance, nil, "mesh")
+		local mesh = Mesh()
+		instance.data.meshes[mesh] = { ntriangles = 0 }
```

Comment:
Make sure spamming these doesn't crash the game. Need to make empty ones consume a few triangles if so.